about.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: docs
+title: About
+headline: All About Pico
+description: What *is* a "stupidly simple, blazing fast, flat file CMS" anyway?
+toc:
+  a-flat-file-cms: A Flat File CMS
+  web-pages-without-html:
+    _title: Web Pages Without HTML
+    markdown-makes-formatting-easy: Markdown Makes Formatting Easy
+    control-your-metadata-with-yaml: Control Your Metadata With YAML
+  flexibility-and-customization:
+    _title: Flexibility and Customization
+    themes: Themes...
+    plugins: Plugins...
+  getting-started: Getting Started
+nav-url: /about/
+nav: 2
+---
+
+Pico is a stupidly simple, blazing fast, flat file CMS.  For those of us who aren't an expert webmaster, what does that even mean?  In the most basic sense, it means that there is no administration backend or database to deal with. You simply create .md files in the content folder and those files become your pages.  There's *much* more to Pico than that though.
```

Comment:
I'm pretty sure that even "expert webmaster's" aren't invariably aware of what this is :smile: